src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -536,11 +536,14 @@ private static async Task ReportExceptionToClient(ServerStore server, Subscripti
                 }
                 else
                 {
-                    connection.AddToStatusDescription("Subscription error");
-
-                    if (connection._logger.IsInfoEnabled)
+                    if (ex is not OperationCanceledException)
                     {
-                        connection._logger.Info("Subscription error", ex);
+                        connection.AddToStatusDescription("Subscription error");
```

Comment:
In this case you are not adding that to the StatusDescription and in the 2nd place you are, please explain me the difference

Reply:
since it was unexpected exception it was logged twice to the `StatusDescription`, now that we made it "expected", I will log it once with "Failed to process subscription" and don't log it as "unexpected" exception.